components/Filter/FilterBy/Collection.tsx

```diff
@@ -0,0 +1,194 @@
+      } as CollectionFilter,
+    },
+    skip: !!selectedCollection,
+    ssr: false,
+  })
+  const collections = collectionData?.collections?.nodes
+
+  const collection = useMemo(() => {
+    if (selectedCollection) return selectedCollection
+    if (!filterResult.collection) {
+      setCollection(undefined)
+      return
+    }
+    const [chainId, address] = filterResult.collection.split('-')
+    if (!chainId || !address) return
+    const collection = collections?.find(
+      (x) => x.address === address && x.chainId === parseInt(chainId, 10),
+    )
+    setCollection(
+      collection ? { chainId: collection.chainId, address } : undefined,
+    )
+    return collection
+  }, [collections, filterResult.collection, selectedCollection, setCollection])
+
+  return collection ? (
+    <AccordionItem>
+      <AccordionButton>
+        <Heading variant="heading2" flex="1" textAlign="left">
+          {selectedCollection
+            ? t('filters.assets.properties.label')
+            : t('filters.assets.properties.labelWithCollection')}
+        </Heading>
+        <AccordionIcon />
+      </AccordionButton>
+      <AccordionPanel>
+        <Stack spacing={3}>
+          {!selectedCollection && (
+            <CollectionListItem
+              width="full"
+              borderColor="brand.500"
+              bgColor="brand.50"
+              borderWidth="1px"
+              borderRadius="md"
+              padding={2}
+              textAlign="left"
+              cursor="pointer"
+              onClick={() => onFilterChange({ collection: null, traits: [] })}
+              collection={convertCollection(collection as any)}
+              closable
+            />
+          )}
+          <InputGroup>
+            <InputLeftElement pointerEvents="none">
+              <Icon as={HiOutlineSearch} w={6} h={6} color="gray.400" />
+            </InputLeftElement>
+            <Input
+              placeholder={t('filters.assets.properties.search.placeholder')}
+              type="search"
+              onChange={(e) => setPropertySearch(e.target.value)}
+            />
+          </InputGroup>
+        </Stack>
+      </AccordionPanel>
+    </AccordionItem>
+  ) : (
+    <AccordionItem>
+      <AccordionButton>
+        <Heading variant="heading2" flex="1" textAlign="left">
+          {t('filters.assets.collections.label')}
+        </Heading>
+        <AccordionIcon />
+      </AccordionButton>
+      <AccordionPanel>
+        <Stack spacing={4}>
+          <InputGroup>
+            <InputLeftElement pointerEvents="none">
+              <Icon as={HiOutlineSearch} w={6} h={6} color="gray.400" />
+            </InputLeftElement>
+            <Input
+              placeholder={t('filters.assets.collections.search.placeholder')}
+              type="search"
+              onChange={(e) => setCollectionSearch(e.target.value)}
+            />
+          </InputGroup>
+          <List>
+            {!collections ? (
+              new Array(4)
```

Comment:
Limit in the query is set to 8, skeleton should be the same:
```suggestion
              new Array(8)
```

Reply:
[Change empty state array lengths](https://github.com/liteflow-labs/starter-kit/pull/375/commits/2276be0a80a95fc5c75325b609511557271e6333)